rails/blueprints/letter_opener/template.rb

```diff
@@ -0,0 +1,13 @@
+def apply_self!
+    if yes?("Would you like to install Letter Opener?", :blue)
+      gem_group :development do
+        gem "letter_opener"
+      end
+      run "bundle install"
+      insert_into_file 'config/environments/development.rb', "\n\tconfig.action_mailer.perform_deliveries = true", after: "config.action_mailer.delivery_method = :letter_opener"
```

Comment:
this works, but you can also just do `environment "config.action_mailer.perform_deliveries = true", env: 'development'` and it will handle tabs / afters and whatnot.